src/layout/Input/types.d.ts

```diff
@@ -2,4 +2,5 @@ import type { IInputFormatting, ILayoutCompBase, ILayoutCompWillBeSavedWhileTypi
 
 export interface ILayoutCompInput extends ILayoutCompBase<'Input'>, ILayoutCompWillBeSavedWhileTyping {
   formatting?: IInputFormatting;
+  searchField?: boolean;
```

Comment:
I believe this might be a bit restrictive - we can't add more variants in the future without having to define rules for which boolean field gets checked first. How about a `variant?: 'text' | 'search'` instead, where `text` is the default variant?